.gitignore

```diff
@@ -6,4 +6,5 @@ node_modules
 replacements.*.json.tmp
 
 /build
+dist
```

Comment:
I would put everything into /build folder by creating more nesting there. having `build` and `dist` folders at the root of the project is confusing.

Reply:
Yea I will move it to build/src as you mentioned on telegram